tests/pytests/pkg/integration/test_salt_ufw.py

```diff
@@ -3,17 +3,41 @@
 import pytest
 
 
+@pytest.fixture
+def salt_systemd_setup(
+    salt_call_cli,
+    install_salt,
+):
+    """
+    Fixture to set systemd for salt packages to enabled and active
+    Note: assumes Salt packages already installed
+    """
+    install_salt.install()
+
+    # ensure known state, enabled and active
+    test_list = ["salt-api", "salt-minion", "salt-master"]
+    for test_item in test_list:
+        test_cmd = f"systemctl enable {test_item}"
+        ret = salt_call_cli.run("--local", "cmd.run", test_cmd)
+        assert ret.returncode == 0
+
+        test_cmd = f"systemctl restart {test_item}"
+        ret = salt_call_cli.run("--local", "cmd.run", test_cmd)
+        assert ret.returncode == 0
+
+
 @pytest.mark.skip_on_windows
```

Comment:
Skip for MacOS also, as there is no SystemD

Reply:
You are forgetting what I said in Standp, about the package tests not working due to Caleb getting cut, and in order to move my PR along, skip them and have them fixed later, since unrelated to my PR fixes and tests, and I need to get my PR into the next minor release,  See https://github.com/saltstack/salt/issues/66672 which is Tech Debt to fix these broken tests.  Already tried fixing them for a couple of days and failed, hence they need more time.  Those tests also need to be re-architect'd, (imho), to test the actual package installed and not the pytest factory implementations.